### DISCUSSION TITLE
Hi, could you provide the source code of stochastic elliptic equation in the paper "Quantifying total uncertainty in physics-informed neural networks for solving forward and inverse stochastic problems"?

### DISCUSSION THREAD

**Discussion by thevelvetunderground** [lululxvi/deepxde#934] [Q&A]
Hi, Lu Lu, after reading your paper "Quantifying total uncertainty in physics-informed neural networks for solving forward and inverse stochastic problems" I'm still confused about some theories in your article. Could you please provide me the source code of inverse problem:stochastic elliptic equation?

**Comment by lululxvi** [comment]
You can reach out to the corresponding author.
